test/pytest/test_bridge.py

```diff
@@ -1298,14 +1298,16 @@ async def test_fsinfo_watch_identity_changes(
 
 @pytest.mark.asyncio
 async def test_fsinfo_self_owner(transport: MockTransport, tmp_path: Path) -> None:
-    client = await FsInfoClient.open(transport, tmp_path, ['user', 'uid', 'group', 'gid'])
+    client = await FsInfoClient.open(transport, tmp_path, ['user', 'uid', 'group', 'gid'], fnmatch='')
```

Comment:
I don't understand this change.  `''` should already be the default value if the attrs array doesn't contain `entries`.

```python
        attrs = set(get_strv(options, 'attrs'))
        self.getattrs = self.make_getattrs(attrs - {'targets', 'entries'})
        self.fnmatch = get_str(options, 'fnmatch', '*' if 'entries' in attrs else '')
```

Reply:
That comes from the test FsInfoClient `            fnmatch: str = '*.txt',`